cmd/loadtest/loadtest.go

```diff
@@ -163,6 +217,42 @@ func initializeLoadTestParams(ctx context.Context, c *ethclient.Client) error {
 	}
 	inputLoadTestParams.CurrentBaseFee = header.BaseFee
 
+	modes := *inputLoadTestParams.Modes
+	if len(modes) == 0 {
+		return fmt.Errorf("expected at least one mode")
+	}
+
+	inputLoadTestParams.ParsedModes = make([]loadTestMode, 0)
+	for _, m := range modes {
+		parsedMode, err := characterToLoadTestMode(m)
+		if err != nil {
+			return err
+		}
+		inputLoadTestParams.ParsedModes = append(inputLoadTestParams.ParsedModes, parsedMode)
+	}
+
+	if len(modes) > 1 {
+		inputLoadTestParams.MultiMode = true
+	} else {
+		inputLoadTestParams.MultiMode = false
+		inputLoadTestParams.Mode, _ = characterToLoadTestMode((*inputLoadTestParams.Modes)[0])
+	}
+
+	if hasMode(loadTestModeRandom, inputLoadTestParams.ParsedModes) && inputLoadTestParams.MultiMode {
+		return fmt.Errorf("random mode can't be used in combinations with any other modes")
```

Comment:
seems to conflict with previous comment that its only modes after random that don't take effect